exporter.go

```diff
@@ -273,8 +273,8 @@ func main() {
 		app        = kingpin.New("windows_exporter", "")
 		configFile = app.Flag(
 			"config.file",
-			"YAML configuration file to use. Values set in this file will override flags.",
-		).Default(os.ExpandEnv("$ProgramFiles\\windows_exporter\\config.yml")).String()
```

Comment:
There's a note in the readme about the default file that we need to remove as well

Reply:
Good spot, I'll fix that up